cmd/goProbe/config/config.go

```diff
@@ -169,18 +162,23 @@ func (c CaptureConfig) validate() error {
 
 func (i Ifaces) validate() error {
 	if len(i) == 0 {
-		return fmt.Errorf("No interfaces were specified")
+		return fmt.Errorf("no interfaces were specified")
 	}
 
 	for iface, cc := range i {
 		err := cc.validate()
 		if err != nil {
-			return fmt.Errorf("Interface '%s' has invalid configuration: %s", iface, err)
+			return fmt.Errorf("%s: %w", iface, err)
 		}
 	}
 	return nil
 }
 
+// Validate validates the interfaces configuration
+func (i Ifaces) Validate() error {
+	return i.validate()
```

Comment:
Stupid question, but since it just wraps the private method it's probably easier to just make the private one public, no? :stuck_out_tongue:

Reply:
Not really because that wouldn't play together with the `validator` approach used to validate the config file.

Reply:
Mh, well, you could make the interface method public, too (if it's going to be exposed anyway there should be no harm I guess). But it's nitpicking of course...